browser/ui/webui/brave_rewards_ui.cc

```diff
@@ -387,9 +387,17 @@ void RewardsDOMHandler::OnGrantCaptcha(
 }
 
 void RewardsDOMHandler::GetGrantCaptcha(const base::ListValue* args) {
+#if defined(OS_ANDROID)
+  // TODO(samartnik): we need different call from JS, currently using this one to make sure it all work
+  // As soon as @ryanml adds separate action for safetynet, we will move that code
+  if (rewards_service_) {
```

Comment:
nit: can we move rewards service check at the beginning, so that we are not duplicating code.

Reply:
thx